src/legacy/core_plugins/embeddable_api/public/panel/_embeddable_panel.scss

```diff
@@ -0,0 +1,198 @@
+/**
+  * EDITING MODE
+  * Use .dshLayout--editing to target editing state because
+  * .embPanel--editing doesn't get updating without a hard refresh
+  */
+
+$dshEditingModeHoverColor: transparentize($euiColorWarning, lightOrDarkTheme(.9, .7));
+
+.embPanel {
```

Comment:
There now seems to be two files with the same panel styles, this one and https://github.com/elastic/kibana/blob/7a91eb808bc8d4bb8e0443ce85c401b366a971a7/src/legacy/core_plugins/dashboard_embeddable/public/embeddable/panel/_dashboard_panel.scss

Does there need to be two separate selectors that behave the same or can one be the base styles and the other apply a second class in the case of special styles?

Reply:
cleaned up!  no more dshPanel, only needed embPanel